tests/components/cert_expiry/test_sensors.py

```diff
@@ -15,8 +17,8 @@
 from tests.common import MockConfigEntry, async_fire_time_changed
 
 
-@patch("homeassistant.util.dt.utcnow", return_value=static_datetime())
-async def test_async_setup_entry(mock_now, hass: HomeAssistant) -> None:
```

Comment:
What is the mock_now?

Reply:
This is what the patch decorator injects

Reply:
It isn't used anywhere in the function

Reply:
aah, I thought it might be a fixture. not sure where it would be used, but if this was a local one it could be removed

Reply:
You might be able to use the `freezer: FrozenDateTimeFactory` fixture here